names.adoc

```diff
@@ -0,0 +1,184 @@
+= Jakarta EE Project Names and Codes
+:projectBaseURL: https://projects.eclipse.org/projects
+:gitBaseURL: https://github.com/eclipse-ee4j
+
+|===
+|Project Id | Project Name | Specification | Short Name
```

Comment:
The last column needs to be "Code", or the places where this column is referenced in operations.adoc need to refer to it as "short name".

Reply:
I changed it to code. I think short name is better, but I this might not be the best time to discuss.

Reply:
I agree on both counts!  :-)